api.planx.uk/modules/webhooks/service/sendNotification/index.ts

```diff
@@ -23,7 +23,40 @@ export const sendSlackNotification = async (
   if (disability) message += " [Exempt]";
   if (resubmission) message += " [Resubmission]";
 
-  await slack.send(":incoming_envelope: " + message);
+  // Prefix message with a custom emoji if this submission fits November 2024 ODP Pilot testing criteria
+  const pilotCouncils = [
+    "barnet",
+    "buckinghamshire",
+    "camden",
+    "lambeth",
+    "medway",
+  ];
+  // Message app types are a bit messy - Uniform won't have app type at all so we id by system because only accepts LDCs,
+  //   BOPS will have internal ID with app type prefix, Email & S3 will have full PlanX service name
+  const pilotServices = [
+    "uniform",
+    "happ",
+    "ldc",
+    "apply for planning permission",
+    "apply for a lawful development certificate",
+  ];
+
+  let isPilotEvent = false;
+  pilotCouncils.forEach((council) => {
+    pilotServices.forEach((service) => {
+      if (
+        message?.toLowerCase()?.includes(council) &&
+        message?.toLowerCase()?.includes(service)
+      ) {
+        isPilotEvent = true;
+      }
+    });
+  });
```

Comment:
**Examples of live messages that _should_ meet pilot criteria:**
- New Uniform submission 62725ee7-db0f-4e22-b4e5-3885115154cd [Lambeth]
- New BOPS submission 24-00213-LDCP [https://camden.bops.services/api/v2/planning_applications]
- New S3 + Power Automate submission "Apply for a lawful development certificate" 0d343b34-39aa-451a-ae5e-6aa805e6320e [barnet]

**Examples that should _not_:**
- New S3 + Power Automate submission "Report a planning breach" 0d343b34-39aa-451a-ae5e-6aa805e6320e [barnet]
- New email submission "Apply for building regulations applications" d62ec563-35ef-4399-aef0-29bff9c94afe [buckinghamshire]
-  New email submission "Pre application advice" 9ef9857e-eed4-4db3-b71e-da0595290c4b [doncaster]

Reply:
These notifications are handled via Hasura `Event triggers` _per_ send destination audit table - so you can do submissions & inspect logs via Hasura Console like screenshot below. BUT we only post to Slack in the production environment (not pizzas or staging to avoid noise), so `Response` "message" is just going to say "Skipping notification..."
![Screenshot from 2024-11-13 13-40-42](https://github.com/user-attachments/assets/5fefe66f-054e-4ebf-a61e-b96c9feada4f)

Slack notifications are a bit of a superfluous feature (purely internal convenience & visibility for our own team from pre-Submissions Log days, no logical or functional dependencies) that can be fixed forward IMO !